From 9b5addecd864079b4d9938ef7ff4fade814ca0ea Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Fri, 4 Jul 2014 11:29:06 +0100
Subject: [PATCH 7/8] osmesa is not a module

Well, maybe it is, but it violates a lot of assumptions cygport makes about
modules (for e.g., it can be statically linked to as well), so it's simplest
just to not build it as a module

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 src/mesa/drivers/osmesa/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mesa/drivers/osmesa/Makefile.am b/src/mesa/drivers/osmesa/Makefile.am
index 4157792..610695f 100644
--- a/src/mesa/drivers/osmesa/Makefile.am
+++ b/src/mesa/drivers/osmesa/Makefile.am
@@ -36,7 +36,6 @@ nodist_EXTRA_lib@OSMESA_LIB@_la_SOURCES = dummy.cpp
 lib@OSMESA_LIB@_la_SOURCES = osmesa.c
 
 lib@OSMESA_LIB@_la_LDFLAGS = \
-	-module \
 	-no-undefined \
 	-version-number @OSMESA_VERSION@ \
 	$(GC_SECTIONS) \
-- 
1.8.5.5

